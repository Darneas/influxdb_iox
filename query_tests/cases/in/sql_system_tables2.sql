-- IOX_SETUP: TwoMeasurementsManyFieldsTwoChunks

--
-- system tables reflect the state of chunks, so don't run them
-- with different chunk configurations.
--


-- ensures the tables / plumbing are hooked up (so no need to
-- test timestamps, etc)
-- IOX_COMPARE: sorted
SELECT partition_key, table_name, column_name, storage, row_count, null_count, min_value, max_value, memory_bytes from system.chunk_columns;
