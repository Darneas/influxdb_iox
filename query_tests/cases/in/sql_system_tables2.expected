-- Test Setup: TwoMeasurementsManyFieldsTwoChunks
-- SQL: SELECT partition_key, table_name, column_name, storage, row_count, null_count, min_value, max_value, memory_bytes from system.chunk_columns;
-- Results After Sorting
+---------------+------------+-------------+-------------------+-----------+------------+-----------+-----------+--------------+
| partition_key | table_name | column_name | storage           | row_count | null_count | min_value | max_value | memory_bytes |
+---------------+------------+-------------+-------------------+-----------+------------+-----------+-----------+--------------+
| 1970-01-01T00 | h2o        | city        | OpenMutableBuffer | 1         | 0          | Boston    | Boston    | 309          |
| 1970-01-01T00 | h2o        | city        | ReadBuffer        | 2         | 0          | Boston    | Boston    | 359          |
| 1970-01-01T00 | h2o        | other_temp  | OpenMutableBuffer | 1         | 0          | 72.4      | 72.4      | 297          |
| 1970-01-01T00 | h2o        | other_temp  | ReadBuffer        | 2         | 1          | 70.4      | 70.4      | 471          |
| 1970-01-01T00 | h2o        | state       | OpenMutableBuffer | 1         | 0          | CA        | CA        | 309          |
| 1970-01-01T00 | h2o        | state       | ReadBuffer        | 2         | 0          | MA        | MA        | 347          |
| 1970-01-01T00 | h2o        | temp        | ReadBuffer        | 2         | 1          | 70.4      | 70.4      | 471          |
| 1970-01-01T00 | h2o        | time        | OpenMutableBuffer | 1         | 0          | 350       | 350       | 297          |
| 1970-01-01T00 | h2o        | time        | ReadBuffer        | 2         | 0          | 50        | 250       | 110          |
| 1970-01-01T00 | o2         | city        | OpenMutableBuffer | 2         | 1          | Boston    | Boston    | 309          |
| 1970-01-01T00 | o2         | reading     | OpenMutableBuffer | 2         | 1          | 51        | 51        | 297          |
| 1970-01-01T00 | o2         | state       | OpenMutableBuffer | 2         | 0          | CA        | MA        | 313          |
| 1970-01-01T00 | o2         | temp        | OpenMutableBuffer | 2         | 0          | 53.4      | 79        | 297          |
| 1970-01-01T00 | o2         | time        | OpenMutableBuffer | 2         | 0          | 50        | 300       | 297          |
+---------------+------------+-------------+-------------------+-----------+------------+-----------+-----------+--------------+
